src/components/common/GridList.tsx

```diff
@@ -0,0 +1,14 @@
+import { Grid, GridProps } from '@chakra-ui/react';
+
+const GridList = ({ children, gap = 0 }: GridProps) => {
+  return (
+    <Grid
```

Comment:
```suggestion
const GridList = ({ children, gap = 0, ...props }: GridProps) => {
  return (
    <Grid
      {...props}
```

어떤 속성도 받을 수 있게 확정성 있게 처리하면 좋을 것 같습니다!!